modules/importing-a-data-science-pipeline.adoc

```diff
@@ -4,10 +4,9 @@
 = Importing a data science pipeline
 
 [role='_abstract']
-To help you begin working with data science pipelines in {productname-short}, you can import a YAML file containing your pipeline's code to an active pipeline server. This file contains a Kubeflow pipeline compiled with the Tekton compiler. After you have imported the pipeline to a pipeline server, you can execute the pipeline by creating a pipeline run.
+To help you begin working with data science pipelines in {productname-short}, you can import a YAML file containing your pipeline's code to an active pipeline server, or you can import the YAML file from a URL. This file contains a Kubeflow pipeline compiled by using the Kubeflow compiler. After you have imported the pipeline to a pipeline server, you can execute the pipeline by creating a pipeline run.
```

Comment:
compiled by the Kubeflow compiler (remove the word "using")
apologies if I recommended adding "using" before